build/build-image/Dockerfile

```diff
@@ -55,3 +55,8 @@ ADD rsyncd.password /
 RUN chmod a+r /rsyncd.password
 ADD rsyncd.sh /
 RUN chmod a+rx /rsyncd.sh
+
+# Enable fips build
+ENV GOEXPERIMENT=boringcrypto
```

Comment:
will this require fips for all k8s clients in all DCs?

Reply:
No , we don't do fips only.
So we'll be fips compatible everywhere.
We'll set the cipher to be the fips cipher and client should negotiate the right one, and be fips if fips or simply use the default tls implem.